src/Search.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
You used double quotes on import here, but in the previous file you used single quotes. Be consistent.